### ISSUE TITLE
Scheduled publishing appears to be broken

### ISSUE THREAD

**Issue by bleroy** [bleroy/OrchardImportTest#137]
REPRO:  

  1. Add a new page 
  2. Fill in the details, e.g. title, slug, content, tags, [x] Add to menu, menu text 
  3. Select the "Publish Later" radio button 
  4. Type in a date and time, e.g. 1 minute from the current time 
  5. Click Save 
  6. Navigate to the front-end and hit refresh after the elapsed time passes   

EXPECTED:  
New menu item to appear and page to be published  
  
ACTUAL:  
Page never gets published, and appears indefinitely scheduled in the "Manage
Pages" list  

Work Item Details
--------------------
**Original CodePlex Issue:**	[Issue 16247](http://orchard.codeplex.com/workitem/16247)
**Status:**	Closed
**Reason Closed:**	Unassigned
**Assigned to:**	Unassigned
**Reported on:**	Feb 24, 2010 at 8:27 AM
**Reported by:**	bradmi
**Updated on:**	Jul 7, 2011 at 3:15 AM
**Updated by:**	rpaquay
**Closed on:**	Dec 9, 2010 at 8:15 AM
**Closed by:**	JilkaBian
**Status:**	Closed
**Reason Closed:**	Unassigned
**Assigned to:**	Unassigned
**Reported on:**	Feb 24, 2010 at 8:27 AM
**Reported by:**	bradmi
**Updated on:**	Jul 7, 2011 at 3:15 AM
**Updated by:**	rpaquay
**Closed on:**	Dec 9, 2010 at 8:15 AM
**Closed by:**	JilkaBian

**Comment by bleroy** [comment]
On *2010-02-25 03:31:48 UTC*, **BertrandLeRoy** commented:

Need confirmation of repro. Scheduling is on 5 minute interval.

**Comment by bleroy** [comment]
On *2010-03-10 11:36:07 UTC*, **rdaddis** commented:

changeset #944 (5393fcf261f0) - my experience was slightly different. I created 3 pages: Publish Later, Publish Later 1, and Publish Later 1 (duplicate).


I found the following problems in the process:

A duplicate slug will be allowed when the Publish Setting is set to"Publish Later".
Once the pages are published, the Manage Menu view shows both duplicate pages with the same URL.
Once the pages are published, the Manage Menu view shows all three of these pages with the same Position.
Publish Later & Publish Later 1 did appear on the main menu 5 minutes after the scheduled publishing time. However, both menu links did temporarily throw HTTP 404 errors: "The resource cannot be found". I closed the app & launched it again
 an hour later & these menu links worked properly. 

Next, I added Publish Later 3 and Publish Later 4 pages to see if the HTTP 404 errors were due to caching issues with the browser or ASP.NET Dev Server. Both pages appeared on the main menu a few minutes after the prescribed publishing time, and both page's
 related records were automatically deleted from the Scheduling_ScheduledTaskRecord table in SQL Server. However, I was still getting the HTTP 404 messages. I cleared the browser cache, stopped debugging, & re-launched the application. Then both pages routed
 properly. 

Problems 1 - 3 are clearly bugs, Problem 4 may be just a caching issue.
